.ddev/commands/web/openmage-install

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+## Description: Install OpenMage
+## Usage: openmage-install
+## Example: ddev openmage-install -d -s -k -q
+
+ROOT="${PWD}"
+
+QUIET_INSTALL_FLAG=''
+SAMPLE_DATA_FLAG=''
+SAMPLE_DATA_KEEP_FLAG=''
+USE_DEFAULT_FLAG=''
+
+while getopts 'dskq' flag; do
+  case "${flag}" in
+    d) USE_DEFAULT_FLAG='true' ;;
+    s) SAMPLE_DATA_FLAG='true' ;;
+    k) SAMPLE_DATA_KEEP_FLAG='true' ;;
+    q) QUIET_INSTALL_FLAG='true' ;;
+  esac
+done
+
+
+#quiet install
+if [[ "${QUIET_INSTALL_FLAG}" ]]; then
+  USE_DEFAULT_FLAG='true'
+  SAMPLE_DATA_FLAG='true'
+fi
+
+LOCALXML="${ROOT}/app/etc/local.xml"
+if [ -f "${LOCALXML}" ]; then
+  if [[ "${QUIET_INSTALL_FLAG}" ]]; then
+    DELETE='y'
+  else
+    read -r -p "OpenMage is already installed. Delete local.xml and drop database [y/N]: " DELETE
+    DELETE=${DELETE,,} # to lower
```

Comment:
"OpenMage is already installed. Delete the local.xml file and drop the database? [y/N] "